packages/context/src/binding.ts

```diff
@@ -506,6 +510,24 @@ export class Binding<T = BoundValue> {
     return this;
   }
 
+  /**
+   * Bind the key to an alias of another binding
+   * @param keyWithPath Target binding key with optional path,
+   * such as `servers.RestServer.options#apiExplorer`
+   */
+  toAlias(keyWithPath: BindingAddress<T>) {
+    /* istanbul ignore if */
+    if (debug.enabled) {
```

Comment:
Do we need this check? AFAICT, the debug statement below is not using any expensive computation.

Reply:
We use it consistently in this file with `/* istanbul ignore if */` so that test coverage is not dragged down by the `debug` statement.